script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -83,47 +83,47 @@ a hand of unparalleled heat is unleashed towards Arcueid, melting the rubble in
 <Page24>
 "Arcueid―――!"
 In one instant.
-The tides of battle get turned in one instant.
+The tides of battle are reversed in an instant.
 <Page25>
-Arcueid said that she could withstand temperatures of up to 3000< |o>C.
-However, the 'hand' that melted the rubble with ease had a temperature of well over 3000< |o>C.
+Arcueid said that she could withstand temperatures of up to 3000℃.
+However, the 'hand' that melted the rubble with ease had a temperature well over that.
 <Page26>
-That combined with the other reasons for that dreadful feeling means that not even ashes will remain. Without an escape path, the flames became a tongue that torment its prey.
-The white figure burned away without a trace.
-―――That's what would have occurred if this happened last night.
+That combined with the other reasons for the dreadful feeling means that not even ashes will remain. Without an escape path, the flames become a tongue that torments its prey.
```

Comment:
He had easily managed to accumulate three times that much power. The flames become a tormenting tongue, playing with their trapped prey.

Reply:
I talked to two different translators about それを三つも重ねられた to get a better translation but "That combined with the other reasons for the dreadful feeling" is obviously wrong and I don't even know how that conclusion was reached. Moreover, 重ねられた is probably in the potential form, not the passive form "was able to" rather than "had been".

This isn't an overtranslation, it's just wrong. I stand by that edit. The original is crazy wrong.